packages/inter-protocol/src/psm/psm.js

```diff
@@ -94,18 +93,23 @@ export const start = async (zcf, privateArgs, baggage) => {
   const emptyStable = AmountMath.makeEmpty(stableBrand);
   const emptyAnchor = AmountMath.makeEmpty(anchorBrand);
 
-  const { augmentVirtualPublicFacet, makeVirtualGovernorFacet, params } =
-    await handleParamGovernance(
-      zcf,
-      privateArgs.initialPoserInvitation,
-      {
-        GiveStableFee: ParamTypes.RATIO,
-        MintLimit: ParamTypes.AMOUNT,
-        WantStableFee: ParamTypes.RATIO,
-      },
-      privateArgs.storageNode,
-      privateArgs.marshaller,
-    );
+  const {
+    extendPublicAPI,
+    extendPublicFacet,
+    extendCreatorFacet,
```

Comment:
extend seems like a verb. I'd prefer this be a noun like 'mixin'.  'additionalBehavior' (& `additionalTypes`) would also work for me to indicate that it's behavior to be added to something.

Reply:
`publicFacetMixin`?